res/css/views/rooms/_EventTile.scss

```diff
@@ -1091,15 +1091,25 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
         // Override :not([data-layout="bubble"])
         &[data-layout=group] {
             --MatrixChat_useCompactLayout_group-padding-top: $spacing-4;
+            --MatrixChat_useCompactLayout-top-avatar: 2px;
+            --MatrixChat_useCompactLayout-top-e2eIcon: 3px;
+            --MatrixChat_useCompactLayout_line-spacing-block: 0px;
```

Comment:
This might look redundant but I think this should explicitly ensure that the same (zero) margin/padding is applied to e2eIcon, avatar, line, and reply.